include/glow/Base/Tensor.h

```diff
@@ -761,8 +765,11 @@ template <class ElemTy> class Handle final {
     return std::all_of(begin(), end(), [=](ElemTy e) { return e == trueZero; });
   }
 
-  void dump(llvm::raw_ostream &os) const { dumpImpl(tensor_, os); }
-  void dump() const { dumpImpl(tensor_); }
+  void dump(llvm::raw_ostream &os, unsigned maxNumElem = MAX_DUMP_ELEMS) const {
+    dumpImpl(tensor_, os, maxNumElem);
+  }
+  void dump(unsigned maxNumElem) const { dumpImpl(tensor_, maxNumElem); }
+  void dump() const { dumpImpl(tensor_, MAX_DUMP_ELEMS); }
```

Comment:
Why do we need a separate `dump()`? Wouldn't it be easier to have `dump(unsigned maxNumElem = MAX_DUMP_ELEMS)`?

Reply:
Yeah, so I did this because I'm never sure how visibility of default parameters works in the debugger and I know some people really like to dump tensors from the debugger.

Reply:
Yeah, in the worst case one would need to call `mytensor.dump(100)` in the debugger. But we can also keep the version without arguments around for the sake of convenience. It is up to you.